varcode/vcf.py

```diff
@@ -296,6 +301,9 @@ def dataframes_to_variant_collection(
 
     variant_collection_kwargs : dict, optional
         Additional keyword parameters to pass to VariantCollection.__init__.
+
+    sources : set or list
```

Comment:
Ah forgot about this. Curious what you think about the presence of `sources_to_metadata_dict` on `EffectCollection`. It seems confusing since the metadata dictionaries are still keyed by `Variant`. Maybe the base `Collection` class should only know about a set of sources and only `VariantCollection` has a full `sources_to_metadata_dict`?